CallKitDemo/ProviderDelegate.swift

```diff
@@ -226,4 +226,13 @@ final class ProviderDelegate: NSObject, CXProviderDelegate {
         answerCall = nil
         callManager.removeAllCalls()
     }
+    
+    func configureAudioSession() {
+        // See https://forums.developer.apple.com/thread/64544
+        let session = AVAudioSession.sharedInstance()
+        try? session.setCategory(AVAudioSessionCategoryPlayAndRecord)
+        try? session.setMode(AVAudioSessionModeVoiceChat)
+        try? session.setPreferredSampleRate(44100.0)
+        try? session.setPreferredIOBufferDuration(0.005)
+    }
```

Comment:
Since the audiodevice is a singleton, if we call `setupAudioSession` before the `didActivate` callback, the audio session won't be reconfigured at that point.